### ISSUE TITLE
Documentation request.

### ISSUE THREAD

**Issue by EricEisaman** [seksity/NodeJS-MQTT-SocketIO#1]
Great idea! Could you expand your README.md ?  Does the project work for you?  Is this the most current code?

**Comment by seksity** [comment]
Yes, it currently code and works fine, for README.md may I will update later.
